tests/core/test_load.py

```diff
@@ -135,6 +135,17 @@ def test_load_spatialdata(get_small_sdata):
     assert artifact._local_filepath.resolve() == artifact.cache() == artifact.path
 
 
+def load_blobs__repr__():
+    example_blobs_sdata = ln.core.datasets.spatialdata_blobs()
+    blobs_af = ln.Artifact.from_spatialdata(
+        example_blobs_sdata, key="example_blobs.zarr"
+    ).save()
+    example_blobs_sdata = ln.Artifact.get(key="example_blobs.zarr")
```

Comment:
But this doesn't have `_memory_rep` anymore. So this doesn't actually test the change i think.